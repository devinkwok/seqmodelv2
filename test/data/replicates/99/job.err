non-relevant line
non-relevant line
