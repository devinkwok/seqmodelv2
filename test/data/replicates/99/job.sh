
## activate python environment


## run model
python seqmodel/run.py --init_version=0.0.0 --init_task=ptmask --lr=0.0001 --activate_env_command='' --deactivate_env_command='' --job_author=test-script --gpus=0 --limit_val_batches=10.0 --deterministic=True --check_val_every_n_epoch=10 --default_root_dir=out/v=0.0.0_t=ptmask_/default_data/default_model/lr=1.00E-04/19

## deactivate python environment


# Job script auto-generated by test-script using ShellJob v0.0.0 on 2021-06-12 16:23:27.033769.
