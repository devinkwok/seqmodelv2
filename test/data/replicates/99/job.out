non-relevant line
non-relevant line

[2021-06-12 16:23:28,231] [INFO] - seqmodel/run.py started.
[2021-06-12 16:23:28,231] [INFO] - seqmodel/run.py finished.
